recipes/default.rb

```diff
@@ -18,4 +18,5 @@
   include_recipe 'base2::users'
   include_recipe 'base2::environment'
   include_recipe 'base2::nrpe'
+  include_recipe 'base2::aws_cw_metrics'
```

Comment:
@aaronwalker 
`['base2']['aws']['cw-metrics-enabled']` attribute added to optionally include `aws_cw_metrics` recipe